packages/core/src/common/classes.ts

```diff
@@ -35,6 +35,7 @@ export const INTERACTIVE = `${NS}-interactive`;
 export const LARGE = `${NS}-large`;
 export const LOADING = `${NS}-loading`;
 export const MINIMAL = `${NS}-minimal`;
+export const OUTLINE = `${NS}-outline`;
```

Comment:
nit: if we're going with "outline" (instead of "ghost" or something) then I think we should make this word an adjective: "outlined". then it will match up with the other modifiers in this list